plsync/mlabconfig.py

```diff
@@ -172,17 +189,18 @@ def export_experiment_records_v4(output, sites, experiment, decoration=''):
 
 
 def export_experiment_records_v6(output, sites, experiment, decoration=''):
-    comment(output, '%s v6%s' % (experiment.dnsname(), (
-        ' decorated' if decoration else '')))
+    comment(output, '%s v6%s' % (experiment.dnsname(), (' decorated' if
+                                                        decoration else '')))
     for site in sites:
         # TODO: change site['nodes'] to a pre-sorted list type.
         for node in sorted(site['nodes'].values(), key=lambda n: n.hostname()):
             # TODO: remove sitenames (or exclude mlab4's).
             if (node.ipv6_is_enabled() and experiment.ipv6(node)):
                 write_aaaa_record(output, experiment.sitename(node, decoration),
                                   experiment.ipv6(node))
-                write_aaaa_record(output, experiment.recordname(
-                    node,decoration), experiment.ipv6(node))
+                write_aaaa_record(output, experiment.recordname(node,
+                                                                decoration),
+                                  experiment.ipv6(node))
```

Comment:
This is a philosophical issue that crops up a lot, once auto-formatters get introduced into a codebase. You are completely correct to note that this is a suboptimal layout.  I would have preferred:
```python
write_aaaa_record(output, 
                  experiment.recordname(node, decoration), 
                  experiment.ipv6(node))
```
and you might have preferred a third thing.

However, the overall benefit of using an autoformatter seems to, in practice, far outweigh the cost of the occasional weird line.